completions.py

```diff
@@ -459,7 +459,7 @@ def on_query_completions(self, view, prefix, locations):
 
         # Determine if current file is Lightning
         file_name = view.file_name()
-        is_lightning = True if file_name.split(".")[-1] in ["app", "cmp"] else False
+        is_lightning = True if file_name.split(".")[-1] in ["app", "cmp", "evt"] else False
```

Comment:
@lushang do we need to add any base component in the event?

Reply:
<aura:attribute> may be used in event. So I add in the list to support this situation

Reply:
@lushang , you are right, please pull a new request based on latest version.

Reply:
No problem